templates/terraform.tfvars.openstack-template

```diff
@@ -5,6 +5,7 @@ kubeadm_token = "your-kubeadm-token" # Autogenerated kubeadm token (don't change
 floating_ip_pool = "your-pool-name"
 external_network_uuid = "external-net-uuid" # The uuid of the external network in the OpenStack tenancy
 boot_image = "kubenow-v050"
+disc_format = "raw"
```

Comment:
I should probably change the default format to "qcow2" - for now it's raw while I'm testing.. but I can change this at the end to "qcow2"